src/BannedSymbols.txt

```diff
@@ -0,0 +1 @@
+M:Microsoft.CodeAnalysis.Compilation.GetTypeByMetadataName(System.String); Use WellKnownTypeProvider instead
```

Comment:
After seeing @sharwell's fixes, I think the diagnostic message here should probably have been

> Use GetOrCreateTypeByMetadataName with WellKnownTypeProvider instead

Reply:
Agreed. I will fix it. Thanks @drewnoakes